pagopa/pn-email-templates
# feat(PN-12551): generate html template script

To test
- delete output/templates/*
- run `yarn generate` to check template are correctly produced
